charmstore/lib.py

```diff
@@ -95,12 +87,14 @@ def __init__(self, id=None, api=None, timeout=None):
         self.series = None
         self.maintainer = None
         self.revision = None
+        self.revisions = None
```

Comment:
This overwrites the method and will lead to `NoneType is not callable` errors.

(It's also less than ideal that this is one of the few attributes that is a method rather than a property or simple attribute, but I guess that's the existing interface, so it can't be changed.)

Reply:
self.revisions is _not_ an existing field, it is a new field